frontend/src/main/scala/bloop/engine/tasks/Tasks.scala

```diff
@@ -134,3 +141,102 @@ object CompileTasks {
     state
   }
 }
+
+object RunTasks {
+  def run(state: State, project: Project, className: String, args: Array[String]): State = {
+    val classpath = project.classpath
+    val processConfig = ProcessConfig(project.javaEnv, classpath)
+    val exitCode = processConfig.runMain(className, args, state.logger)
+    val exitStatus = {
+      if (exitCode == ProcessConfig.EXIT_OK) ExitStatus.Ok
+      else ExitStatus.UnexpectedError
+    }
+
+    state.mergeStatus(exitStatus)
+  }
+
+  def findMainClasses(state: State, project: Project): Array[String] = {
```

Comment:
I'm curious: why do we return an array here?

Reply:
You mean as opposed to any other data structure? It's because that's the one we use most often in bloop, but it could be anything else.

Reply:
I thought we were using `Seq` as the default. I've only seen `Array` being used when the sbt test interface API has required it and for `bloop.Project`.